core/services/pipeline/test_helpers.go

```diff
@@ -6,6 +6,26 @@ import (
 	"github.com/jinzhu/gorm"
 )
 
+const (
```

Comment:
I think anything that could be used for both a whitebox and a blackbox test should be exposed inside the package under test. That way you could conceivably write an whitebox integration test across two package boundaries without roping in cltest (which because it imports everything, really needs to strictly be used for blackbox testing in unqiue xxx_test packages)